NurVo_Frontend/src/pages/Login.tsx

```diff
@@ -34,23 +35,27 @@ const MainPage = ({ navigation, route }) => {
   const [userPassword, setUserPassword] = useState('');
   const [isAlret, setIsAlert] = useState(false);
   const [alertMessage, setAlertMessage] = useState('');
+  const { isLogged, setIsLogged } = useContext(UserContext);
 
   // 로그인
   useEffect(() => {
     async function Login() {
+      console.log("login")
       try {
+        console.log("userId: ", userId, "userPassword: ", userPassword);
         const result = await fetchLogin({
           "userId": userId,
           "password": userPassword,
         });
-        if (result === "Invalid username or password") {
+        if (!result) {
```

Comment:
if (!result) {
```suggestion
       if (result === "Invalid username or password") {
```
이 코드로 수정해주세요.

Reply:
저도 그렇게 의도했는데 꼼꼼하지 못했네요ㅠㅠ 감사합니다!